cloudify_awssdk/autoscaling/resources/autoscaling_group.py

```diff
@@ -155,7 +155,7 @@ def create(ctx, iface, resource_config, **_):
 
     subnet_list_string = params.get(SUBNET_LIST)
     subnet_list = \
-        subnet_list_string.split(', ') if \
+        subnet_list_string.split(',') if \
```

Comment:
What do you think about:
```ipython
from re import sub
sub("[^\w]", " ",  s).split()
```

Reply:
I think it will be much better , will update it